integration/kubernetes/k8s-attach-handlers.bats

```diff
@@ -20,6 +20,9 @@ setup() {
 }
 
 @test "Running with postStart and preStop handlers" {
+	wait_time=20
+	sleep_time=2
```

Comment:
Since these values are common, I wonder if it is worth adding them to `lib.sh` or `common.bash` as `k8s_wait_time_secs` and `k8s_sleep_time_secs`?